receiver/mongodbatlasreceiver/README.md

```diff
@@ -34,6 +34,7 @@ In order to collect access logs, the requesting API key needs the appropriate pe
 
 MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/logs/#logs) recommends a polling interval of 5 minutes.
 
+- `base_url` (optional) set to https://cloud.mongodbgov.com to connecto to Atlas Gov
```

Comment:
we should probably describe the parameter purpose here (set the base URL) instead of  using a specific usage example.